examples/csv-io/csv_io.bal

```diff
@@ -81,17 +80,13 @@ public function main() returns @tainted error? {
 
 // Creates a `table` and adds some data.
 function createTableAndAddData() returns table<Employee> {
-    table<Employee> employeeTable = table {};
+    table<Employee> employeeTable = table [];
     Employee[] employees = [];
     employees[0] = {id: "1", name: "Allen", salary: 300000.0};
     employees[1] = {id: "2", name: "Wallace", salary: 200000.0};
     employees[2] = {id: "3", name: "Sheldon", salary: 1000000.0};
     foreach var employee in employees {
         var result = employeeTable.add(employee);
-        if (result is error) {
-            log:printError("Error occurred while adding data to table: ",
-                            err = result);
-        }
```

Comment:
Don't we get any error while adding to the table

Reply:
AFAIK, with the new table syntax, the `add` method doesn't return an error and instead, it panics. See https://github.com/ballerina-platform/ballerina-lang/blob/master/langlib/lang.table/src/main/ballerina/src/lang.table/table.bal#L83

Reply:
ok, then can we mention actual return type instead of var. is it () here?

Reply:
it does not return anything. we can just remove assignment

Reply:
Ack. Updated in a63e083